easybuild/easyconfigs/s/SeqAn3/SeqAn3-3.0.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "Tarball"
+
+name = 'SeqAn3'
+version = "3.0.0"
+
+homepage = 'https://github.com/seqan/seqan3'
+description = """SeqAn is an open source C++ library 
+ of efficient algorithms and data structures for the 
+ analysis of sequences with the focus on biological 
+ data. Our library applies a unique generic design 
+ that guarantees high performance, generality, 
+ extensibility, and integration with other libraries. 
+ SeqAn is easy to use and simplifies the development 
+ of new software tools with a minimal loss of 
+ performance.
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/seqan/seqan3/releases/download/%(version)s/']
+sources = ['seqan-%(version)s-with-submodules.tar.gz']
+checksums = ['ba8f0fd5d19289f95823235b69fdc5a12aba3578292463eecf750722e8662bd3']
+
+files_to_copy = ['include',
+                 'submodules',
+                 'build_system']
+
+sanity_check_paths = {
+    'files': [],
+    'dirs':  ['include', 'submodules']}
+
+modextrapaths = {
+    'INCLUDEPATH': 'include/',
+    'INCLUDEPATH': 'submodules/range-v3',
+    'INCLUDEPATH': 'submodules/sdsl-lite',
+    'INCLUDEPATH': 'submodules/cereal',
```

Comment:
@cmeesters This won't work, you'll only have one of these actually used (and to make it even more fun, it'll be a random one, since keys in Python dictionaries don't have any order...).

You can fix this as follows:

```python
modextrapaths = {
    'INCLUDEPATH': ['include', 'submodules/range-v3', 'submodules/sdsl-lite', 'submodules/cereal'],
}

Reply:
thanks and outch, outch, outch ...

Reply:
done